gm4_combat_expanded/pack.mcmeta

```diff
@@ -0,0 +1,31 @@
+{
+    "pack": {
+        "pack_format": 10,
+        "description": [
+            "Combat Expanded",
+            "\n",
+            {
+                "text": "Gamemode 4 for 1.19",
+                "color": "#4AA0C7"
+            }
+        ]
+    },
+    "module_name": "Combat Expanded",
+    "module_id": "combat_expanded",
+    "libraries": [
+        "lib_forceload",
+        "lib_lore"
+    ],
+    "site_description": "Use special armor and weapon modifiers to defend against mobs that grow ever stronger.",
+    "video_link": "",
+    "wiki_link": "",
+    "required_modules": [],
+    "recommended_modules": [],
```

Comment:
The Zauber compatibility refers to letting custom items be 'zauberfied' (right now Zauber Cauldrons only supports vanilla golden armour), there's no real interaction between the modules otherwise so I don't think it should go in recommended modules.